src/lib/common/tag.h

```diff
@@ -30,6 +30,22 @@
 #include "common/Format.h"
 
 
+// Macros to be used for JSON rendering
+
+#define JSON_PROP(name) std::string("\"" + std::string(name) + "\":")
```

Comment:
As it seems, std::string accepts char* not only on its right but also to its left :-)

NTC